docs/example_usecases.md

```diff
@@ -0,0 +1,15 @@
+<!--
+SPDX-FileCopyrightText: 2023 Springtime authors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+Here, we provide several notebooks to show how to use `springtime` package in
+different use cases. Click on each title to download the notebook.
```

Comment:
I'd humbly suggest to remove this page from the docs.

Reply:
removed!